### PR TITLE
Stories/dt/scoreoutputter #28

### LINKED ISSUE #28: Score outputter

### PR DESCRIPTION
**Pull request by terw-dan**
Loader for the Analyzer to use on a page with window and document. 
Scoreformatter to visualize the score from the analyzer on the page.
fixes #28

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    line-height: 20px;
+}
+
+.inputForm input{
+    width: 400px;
+    line-height: 20px;
+    margin: 5px;
+}
+
+.inputForm textArea{
```
Comment (andizer) [inline css/style.css]: `textArea` should be `textarea`

File: js/analyzer.js [hunk 2]
```diff
@@ -143,24 +160,29 @@ Analyzer.prototype.keywordDensityRating = function(keywordDensity){
...
+    return result;
 };
+
 /**
  * subHeadings checker to check if keyword is present in given headings.
  * @param matches
  * @returns {number}
  */
-Analyzer.prototype.subHeadingsCheck = function(matches){
+Analyzer.prototype.subHeadingsCheck = function( matches ){
```
Comment (andizer) [inline js/analyzer.js]: Codestyle?

File: js/analyzer.js [hunk 3]
```diff
@@ -169,71 +191,70 @@ Analyzer.prototype.subHeadingsCheck = function(matches){
...
-Analyzer.prototype.fleschReading = function(){
-    var score =  (206.835 - (1.015 * (yst_preProcessor.__store.wordcount / yst_preProcessor.__store.sentencecount)) - (84.6 * (yst_preProcessor.__store.syllablecount / yst_preProcessor.__store.wordcount))).toFixed(1);
-    if(score < 0){score = 0;}else if (score > 100){score = 100;}
-    return {name: "fleschReading", result: score};
+Analyzer.prototype.fleschReading = function() {
+    var score =  ( 206.835 -
+        ( 1.015 * ( this.preProcessor.__store.wordcountNoTags / this.preProcessor.__store.sentenceCountNoTags ) ) -
+        ( 84.6 * ( this.preProcessor.__store.syllablecount / this.preProcessor.__store.wordcountNoTags ) ) )
+        .toFixed( 1 );
+    if( score < 0 ){ score = 0; }else if ( score > 100 ){ score = 100; }
```
Comment (andizer) [inline js/analyzer.js]: This line is hard to read

File: js/analyzer.js [hunk 4]
```diff
@@ -241,12 +262,12 @@ Analyzer.prototype.linkCount = function(){
  * @param url
  * @returns {string}
  */
-Analyzer.prototype.linkType = function(url){
+Analyzer.prototype.linkType = function( url ) {
     var linkType = "other";
-    if(url.match(/https?:\/\//g) !== null){
+    if( url.match( /https?:\/\//g ) !== null ){
```
Comment (andizer) [inline js/analyzer.js]: What if I have a http URL? Does the regex catch this also?

**Comment by andizer** [comment]
CR done :+1:

**Comment by Petervw** [comment]
Acceptance :+1:

**Pull request merged**